internals/secrethub/acl_list_test.go

```diff
@@ -40,17 +40,27 @@ func TestACLListCommand_run(t *testing.T) {
 		"client error": {
 			cmd: ACLListCommand{},
 			accessrules: fakeclient.AccessRuleService{
-				Lister: &fakeclient.AccessRuleLister{
-					Err: testError,
+				ListFunc: func(path string, depth int, ancestors bool) ([]*api.AccessRule, error) {
+					return nil, testError
+				},
+			},
+			dirs: fakeclient.DirService{
+				GetTreeFunc: func(path string, depth int, ancestors bool) (*api.Tree, error) {
+					return nil, nil
```

Comment:
Before, there was no function provided to get the tree. Is there a specific reason you changed this? I think the cases are easier to understand if we only set the fields that are used (and leave the other fields to their zero values). What do you think?

Reply:
This was a mistake, fixed it :+1:

Reply:
@Marton6 This was not the only occurrence. There's several additions like this. Could you update the others as well?

Reply:
The others (in this file, at least) are required.

Reply:
Yes, they are indeed in other files. Have pointed at a view. They're easy to spot if you look at the diff. So please go over the other files there as well.